core/src/main/java/org/apache/accumulo/core/dataImpl/KeyExtent.java

```diff
@@ -495,11 +495,12 @@ public Range toDataRange() {
   }
 
   public Range toMetadataRange() {
-    Text metadataPrevRow = new Text(getTableId().getUtf8());
-    metadataPrevRow.append(new byte[] {';'}, 0, 1);
-    if (getPrevEndRow() != null) {
-      metadataPrevRow.append(getPrevEndRow().getBytes(), 0, getPrevEndRow().getLength());
-    }
+    Text metadataPrevRow;
```

Comment:
Another possible way to do this is the following.  It's slightly shorter, however the if stmt may be more readable.  This is a stylistic comment, so feel free to ignore it.

```java
Text metadataPrevRow = TabletsSection.getRow(getTableId(), getPrevEndRow() == null ? EMPTY : getPrevEndRow());
```

Reply:
That's fine.  I like that.